handbook/README.md

```diff
@@ -105,6 +105,28 @@ A browser compatibility check of [fleetdm.com](https://fleetdm.com/) should be c
 - Document any issues in GitHub as a [bug report](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&template=bug-report.md&title=), and assign for fixing.
 - If in doubt about anything regarding design or layout, please reach out to the Design team.
 
+### Fleet docs
+
+#### Adding a link to Fleet docs
+The `fleetdm.com/docs` compilation process does not account for relative links to directories outside of `/docs`.
```

Comment:
Consider starting this section off by clarifying that this _does_ work for links within the docs, and including an example of a correct link to docs.

e.g. 

> You can link documentation pages to each other using relative paths.  For example, in `docs/1-Using-Fleet/1-Fleet-UI.md`, you can link to `docs/1-Using-Fleet/9-Permissions.md` by writing `[permissions](./9-Permissions.md)`.  This will be automatically transformed into the appropriate URL for fleetdm.com/docs.


Also please consider clarifying that linking to other handbook pages from another handbook page works the same way.